physical_property/logP/analysis/logP_predictions/logP-DavyGuan-1.csv

```diff
@@ -75,7 +75,8 @@ The University of Sydney
 # The name must not exceed 40 characters.
 # The 'Name:' keyword is required as shown here.
 Name:
-RayLogP三_QSPR_Mordred2D_TPOT-AutoML
+# RayLogP三_QSPR_Mordred2D_TPOT-AutoML
```

Comment:
I thought this was a name you updated?

Reply:
It's updated, see the line below (line 79)